packages/xstate-react/test/useService.test.tsx

```diff
@@ -54,55 +61,6 @@ describe('useService hook', () => {
     });
   });
 
-  it('service actions should be configurable', () => {
-    const counterService = interpret(counterMachine).start();
-
-    const Counter = () => {
-      const [state, send] = useService(counterService);
-      const [otherState, setOtherState] = useState('');
-
-      useEffect(() => {
-        counterService.execute(state, {
```

Comment:
This test was IMHO incorrect - using this pattern there is a reasonably high chance that this action would **not** be executed. The only way to truly be sure that an action is going to be executed is to put it inline or in an `actions` object.